dnf-behave-tests/dnf/plugins-core/config-manager-opts.feature

```diff
@@ -0,0 +1,371 @@
+      [repo2]
+      skip_if_unavailable=1
+      """
+
+Scenario: repository configuration overrides - globs in repo id, set option more times with different value
+ Given I create directory "/etc/dnf/repos.override.d/"
+   When I execute dnf with args "config-manager setopt r?p*.skip_if_unavailable=1 repo1.priority=50 repo2.skip_if_unavailable=0"
+   Then the exit code is 1
+    And stderr contains "Sets the "skip_if_unavailable" option of the repository "repo2" again with a different value: "1" != "0""
+
+
+Scenario: repository configuration overrides, glob does not match any repo "id"
+ Given I create directory "/etc/dnf/repos.override.d/"
+   When I execute dnf with args "config-manager setopt non*repo.skip_if_unavailable=1"
+   Then the exit code is 1
+    And stderr contains "No matching repository to modify: non\*repo"
+
+
+Scenario: repository configuration overrides - new option, missing destination directory
+   When I execute dnf with args "config-manager setopt repo1.skip_if_unavailable=1"
+   Then the exit code is 1
+    And stderr contains "Directory ".*/etc/dnf/repos.override.d" does not exist. Add "--create-missing-dir" to create missing directories"
+
+
+Scenario: repository configuration overrides - new option, "--create-missing-dir" creates missing destination directory
+   When I execute dnf with args "config-manager setopt repo1.skip_if_unavailable=1 --create-missing-dir"
+   Then the exit code is 0
+    And file "/etc/dnf/repos.override.d/99-config_manager.repo" contents is 
+      """
+      # Generated by dnf5 config-manager.
+      # Do not modify this file manually, use dnf5 config-manager instead.
+      [repo1]
+      skip_if_unavailable=1
+      """
+
+
+Scenario: repository configuration overrides - new option, a file was found instead of destination directory
+ Given I create file "/etc/dnf/repos.override.d" with
+      """
+      """
+   When I execute dnf with args "config-manager setopt repo1.skip_if_unavailable=1"
+   Then the exit code is 1
+    And stderr contains "The path ".*/etc/dnf/repos.override.d" exists, but it is not a directory or a symlink to a directory"
+
+
+Scenario: repository configuration overrides - new option, a symlink to non-existent object was found instead of destination directory
+ Given I create symlink "/etc/dnf/repos.override.d" to file "/non-exist"
+   When I execute dnf with args "config-manager setopt repo1.skip_if_unavailable=1"
+   Then the exit code is 1
+    And stderr contains "The path ".*/etc/dnf/repos.override.d" exists, but it is a symlink to a non-existent object"
+
+
+Scenario: repository configuration overrides - unset/remove option
+ Given I create file "/etc/dnf/repos.override.d/99-config_manager.repo" with
+      """
+      [repo1]
+      enabled=0
+      priority=40
+      skip_if_unavailable=1
+      [repo2]
+      enabled=1
+      priority=50
+      skip_if_unavailable=0
+      """
+   When I execute dnf with args "config-manager unsetopt repo1.priority"
```

Comment:
This use case must be well documented in man pages - config-manager does not touch original repository definition but only  creates or remove overrides